bindings/python/build.rs

```diff
@@ -1,4 +1,3 @@
 fn main() {
     pyo3_build_config::use_pyo3_cfgs();
-    println!("cargo::rustc-check-cfg=cfg(allocator, values(\"default\", \"mimalloc\"))");
```

Comment:
Why did you drop this?

Reply:
If I'm not mistaken, this configuration is not necessary for now